send-transaction-service/src/send_transaction_service.rs

```diff
@@ -311,12 +301,9 @@ impl SendTransactionService {
         result
     }
 
-    fn send_transaction(
-        send_socket: &UdpSocket,
-        tpu_address: &SocketAddr,
-        wire_transaction: &[u8],
-    ) {
-        if let Err(err) = send_socket.send_to(wire_transaction, tpu_address) {
+    fn send_transaction(tpu_address: &SocketAddr, wire_transaction: &[u8]) {
+        let connection = connection_cache::get_connection(tpu_address);
```

Comment:
let's add timings here

Reply:
Putting timing here is not convenient here to get the differences of udp vs quic unless you already know the validator's config. Maybe that belongs to QuicTpuConnection and UdpTpuConnection.

Reply:
I will create a separate issue for this effort

Reply:
Comparing UDP to QUIC isn't the motivation.  I want to know when QUIC is slow and by how much in the context of RPC STS usage

Reply:
Added timing in send_transaction in STS